service/svc.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+ROOTPROJECTPATH="$(
+    cd -- "$(dirname "$0")/.." >/dev/null 2>&1
+    pwd -P
+)"
+
+function install {
+    path=$1
+    if [ -z "$1" ]; then
+        path="/lib/systemd/system"
+    fi
+
+    if [ -e "$path/threeal-bot.service" ]; then
+        echo "Service is already installed"
+        exit
+    fi
+
+    go_path=$(which go)
```

Comment:
ooh right, you make me realized about this.

in case `go_path` is empty, tell to install go (tell this URL for reference: https://go.dev/doc/install), and exit with non zero status (see [this](https://www.gnu.org/software/bash/manual/html_node/Exit-Status.html), i usually just using `1`).